lib/site_prism/exceptions.rb

```diff
@@ -12,6 +12,7 @@ def message
 
   class NoSelectorForElement < StandardError; end
   class TimeoutException < StandardError; end
+  class TimeOutWaitingForExistence < StandardError; end
```

Comment:
Again something I'm slowly trying to do is add in some nice error messages. If you look at `UnsupportedBlock` and design something similar that would be good.

Reply:
I decided to format the message at the point of raising, which I think is what is more standard to do in this case, let me know how I should be formatting the message here. I'm not familiar how to do it here when I want to format two arguments. I was having trouble setting the message when passing arguments to the error constructor.